Whitelist-dApp.md

```diff
@@ -57,6 +57,7 @@ npx hardhat
 ```
 
 Make sure you select `Create a Javascript Project` and then follow the steps in the terminal to complete your Hardhat setup.
+npm install --save-dev @nomicfoundation/hardhat-toolbox
```

Comment:
@yanukadeneth99 I think where he placed this is fine because you install this after you run `npx hardhat`. But @kaustubhthakur , I would say that this should only be done if you're on a Windows Operating System, because this gets installed when you do `npx hardhat` on a MacOS.

Reply:
Yeah, that's true. Where he originally placed it though stays align with how the other tutorials are.